test/Constraints/closures.swift

```diff
@@ -64,19 +64,21 @@ func test13811882() {
 
 
 // <rdar://problem/21544303> QoI: "Unexpected trailing closure" should have a fixit to insert a 'do' statement
+// <https://bugs.swift.org/browse/SR-3671>
 func r21544303() {
-  var inSubcall = true
+  let val = true
+  var inSubcall = val
   {   // expected-error {{expected 'do' keyword to designate a block of statements}} {{3-3=do }}
```

Comment:
I do still want to see what @rintaro thinks; they've spent much more time than me in Parse recently.